services/slack/README.md

```diff
@@ -3,26 +3,96 @@ Slack
 
 [product]: https://slack.com/
 [producti]: http://i.imgur.com/0u1iR8W.png
 
 ## Channels
 
-Slack allows us to create channels to talk to everyone at once, think chat room.
+Slack allows us to create channels to talk to everyone at once, think "chat room".
 
-### General & Random
+### #general & #random
 
-By default slack has two channels, General and Random, and all users are added. General is a place to talk about non work related stuff and Random is a scary place of pure randomness.
+By default slack has two channels, #general and #random, and all users are added. General is a place to talk about work related stuff and Random is a scary place of pure randomness.
+
+### Sparkbox channels
+Official "Sparkbox Sanctioned" channels begin with a `-`. See the note on prefixes below.
+
+##### #-awesomeness
+A channel for sharing awesome notes about coworkers.
+
+##### #-biz
+A place for **bizBot** to keep Sparkbox informed about business development.
+
+##### #-conferences
+The channel for sharing interesting conferences, Call for Speakers, and planning the conference you'd like to attend.
+
+##### #-dayton & #-pittsburgh
+Channels to discuss office-specific happenings, such as lunch plans.
+
+##### #-gems
+A collection of quotations from your Sparkbox community, currated in part by **gem-me-bot**. Share a gem publicly with the `:earth-americas:` reaction, or hide it from the public eye with the `:x:` response.
+
+##### #-know-your-company
+A place to discuss fun facts about coworkers, along with weekly KYC email answers.
+
+##### #-sparkbox-pride
+Share your Sparkbox Pride, generally in photo form.
 
 ### Public Channels
 
-Additional channels have been created which any member of the Sparkbox team can join at any time. These channels cover a range of topics from discussion on design, to video games, to weekly lunches. Any member of the team can start a new channel or join an existing one. There are currently over 150 channels.
+Additional channels have been created which any member of the Sparkbox team can join at any time. These channels cover a range of topics from discussion on design, to video games, to weekly lunches. Any member of the team can start a new channel or join an existing one.
+
+### Creating New Channels
+
+There are obvious times to create a new channel, such as when a new client engagement starts. There are also not as obvious times. When deciding to create a new channel, ask yourself these questions:
+- Is this channel new and not covered by any existing channels? Have I asked in #general if a similar channel exists?
+- Will this channel reduce ongoing unrelated clutter in another channel?
+
+After you create a new channel, be sure to set a clear, concise channel purpose so that other Sparkboxers can easily understand what the channel is discussing. If there isn’t enough time to set the channel topic/purpose, it might not be worth your time to create the channel.
```

Comment:
just a little bit too snarky potentially haha:
> If there isn’t enough time to set the channel topic/purpose, it might not be worth your time to create the channel.